### PR TITLE
Make a contributing.md (closes #102)

### LINKED ISSUE #102: Write a CONTRIBUTING.md
We need to write a CONTRIBUTING.md file to explain how to get involved with the project & organization!

### PR DESCRIPTION
**Pull request by aboutdavid**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- Ensure you are using the latest commit or version of Obl.ong.
+- Make sure the issue hasn't already been posted by searching the issues.
+- Give information about the bug: OS, OS version, Browser Version (if it occured in the browser), ruby version (`ruby -v`), git commit (`git rev-parse --short HEAD`), screenshot (if needed).
+
+Also, please do **not** submit security issues via GitHub issues. Please send an E-mail to team@obl.ong.
+
+## 💻 Setting up a development environment
+
+1. Install the currently supported version Ruby. You can use a tool like [Ruby Version Manager (RVM)](https://rvm.io/) to install ruby. Check the `.ruby-version` file to find it. Usually, it's the latest version, but it never hurts to check.
```
Comment (reesericci) [inline CONTRIBUTING.md:21]: rbenv > rvm

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+2. Install Rails. A lot of times it may come preinstalled. Run `rails -v` to check.
+3. Install [bun.sh](https://bun.sh). You can do this by running `curl -fsSL https://bun.sh/install | bash
+4. Pull the submodules. Run `git submoudle init && git submodule update`
```
Comment (reesericci) [inline CONTRIBUTING.md:24]: git submodule is spelled wrong

**Review by reesericci — CHANGES_REQUESTED** [review]
Overall looks great - some very minor things

File: CONTRIBUTING.md (hunk 1 above)
Comment (aboutdavid) [reply CONTRIBUTING.md]: Fixed, but what's the difference? I don't know too much about ruby, sorry

File: CONTRIBUTING.md (hunk 2 above)
Comment (aboutdavid) [reply CONTRIBUTING.md]: Fixed in [9ccf2e8](https://github.com/obl-ong/admin/pull/148/commits/9ccf2e8544214ed40fd8ac0d73b6800384d3b49f)

File: CONTRIBUTING.md (hunk 1 above)
Comment (reesericci) [reply CONTRIBUTING.md]: rbenv is just a better tool that I will actually support if people have questions.

**Review by reesericci — APPROVED** [review]
LGTM

**Pull request merged**